app/services/ncr/approval_manager.rb

```diff
@@ -50,7 +50,14 @@ def force_approvers(users)
       individuals = users.map do |user|
         proposal.existing_step_for(user) || Steps::Approval.new(user: user)
       end
-      proposal.root_step = Steps::Serial.new(child_steps: individuals)
+      unless proposal.root_step && steps_share_parent?(proposal.root_step, individuals)
```

Comment:
not sure I understand the use case for `steps_share_parent?(proposal.root_step, individuals)` -- perhaps we should have a test for this?